docs/source/faq.rst

```diff
@@ -0,0 +1,80 @@
+Issues & FAQ
+===================================
+
+
+FAQ
+########
+
+Issues 
+########
+
+
+Java JVM DLL not found (specific to Mac)
+**********************
+
+One user reported the following error when installing `pyspi` on a MacBook Air an M2 chip and Catalina OS:
+
+.. code-block::
+
+   OSError: [Errno 0] JVM DLL not found /Library/Java/JavaVirtualMachines/jdk-19.jdk/Contents/Home/lib/libjli.dylib
+
+
+This issue is similar to those reported `here <https://stackoverflow.com/questions/71504214/jvm-dll-not-found-but-i-can-clearly-see-the-file>`_ and `here <https://github.com/jpype-project/jpype/issues/994>`_. This issue can arise from the version of OpenJDK identified as the system default
+
+We recommend following this `helpful tutorial <https://blog.bigoodyssey.com/how-to-manage-multiple-java-version-in-macos-e5421345f6d0>`_ by Chamika Kasun to install `AdoptOpenJDK <https://adoptopenjdk.net/index.html>`_. In a nutshell, here are the steps you should run:
+
+Install homebrew if you don't already have it:
+
+.. code-block::
+
+   $ /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+
+
+Install `jenv` as your Java version manager:
+
+.. code-block:: 
```

Comment:
Hmm these code blocks don't seem to be coming out in the RTD. Does it need a newline after the declaration? E.g.,

```
.. code-block:: 

    $ brew install jenv

Reply:
I added newlines after the declarations for all the ones in `faq.rst` that I had missed -- thanks for pointing that out. It looks fine on my end now, but let me know if the issue still persists with my latest commit!